pyproject.toml

```diff
@@ -27,6 +27,8 @@ dependencies = [
     "scipy==1.11.4",
     "zodb",
     "zeo",
+    "pyqt6",
```

Comment:
I guess this will install `pyqt` version 6 for `pip` installation method both on Linux and macOS. Is this an issue ?

Reply:
It's not, just the TRR is running with version 5, but we can update the TRR code